plugins/flytekit-flyin/flytekitplugins/flyin/utils.py

```diff
@@ -0,0 +1,56 @@
+import importlib
+import os
+import sys
+
+from flytekit.core import utils
+from flytekit.core.context_manager import FlyteContextManager
+from flyteidl.core import literals_pb2 as _literals_pb2
+from flytekit.core.type_engine import TypeEngine
+from flytekit.models import literals as _literal_models
+
+
+def load_module_from_path(module_name, path):
+    """
+    Imports a Python module from a specified file path.
+
+    Args:
+        module_name (str): The name you want to assign to the imported module.
+        path (str): The file system path to the Python file (.py) that contains the module you want to import.
+
+    Returns:
+        module: The imported module.
+
+    Raises:
+        ImportError: If the module cannot be loaded from the provided path, an ImportError is raised.
+    """
+    spec = importlib.util.spec_from_file_location(module_name, path)
+    if spec is not None:
+        module = importlib.util.module_from_spec(spec)
+        sys.modules[module_name] = module
+        spec.loader.exec_module(module)
+        return module
+    else:
+        raise ImportError(f"Module at {path} could not be loaded")
+
+
+def get_task_inputs(task_module_name, task_name, context_working_dir):
+    """
+    Read task input data from inputs.pb for a specific task function and convert it into Python types and structures.
+
+    Args:
+        task_module_name (str): The name of the Python module containing the task function.
+        task_name (str): The name of the task function within the module.
+        context_working_dir (str): The directory path where the input file and module file are located.
+
+    Returns:
+        dict: A dictionary containing the task inputs, converted into Python types and structures.
+    """
+    local_inputs_file = os.path.join(context_working_dir, "inputs.pb")
```

Comment:
isn't there similar code in entrypoint.py?  we should reuse that as much as possible.

Reply:
After looking into entrypoint.py https://github.com/flyteorg/flytekit/blob/master/flytekit/bin/entrypoint.py#L87, I found it's somehow hard to reuse the code since the behavior is a little bit different. Do you have any suggestion? Thank you.

Reply:
i think it's ok to add the utils here. this usage is not found in the core flytekit